conftest.py

```diff
@@ -1,11 +1,18 @@
+import glob
+
 import pytest
 
 
+def _py_files(folder):
+    return glob.glob(folder + "/*.py") + glob.glob(folder + "/*/*.py")
+
+
 collect_ignore = [
     # not a test, but looks like a test
     "scrapy/utils/testsite.py",
 ]
 
+collect_ignore += _py_files("tests/CrawlerProcess")
```

Comment:
You can probably add this as `*_py_files("tests/CrawlerProcess")` to the list above instead, and maybe include a comment in the line of the existing one, with a brief explanation of why these files must be ignored.